src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -4607,7 +4603,7 @@ namespace AirflowNetwork {
                 int compnum = compnum_iter->second;
                 AirflowNetworkLinkageData(count).CompNum = compnum;
 
-                auto &surf = m_state.dataSurface->Surface(MultizoneSurfaceData(count).SurfNum);
+                auto const &surf = m_state.dataSurface->Surface(MultizoneSurfaceData(count).SurfNum);
 
```

Comment:
[src/EnergyPlus/AirflowNetwork/src/Solver.cpp:476]:(style),[knownConditionTrueFalse],Condition 'nodeErrorsFound' is always false
[src/EnergyPlus/AirflowNetwork/src/Solver.cpp:4610]:(style),[constVariable],Variable 'surf' can be declared as reference to const